### ISSUE TITLE
Error: Unable to load collection and Error: Unable to load environment

### ISSUE THREAD

**Issue by naveenkumarmallappa** [a85/POSTMan-Chrome-Extension#4630]
**Describe the bug**
Error: Unable to load collection and Error: Unable to load environment, I got this error when auto run the monitor on every day 5 AM on Asia Pacific Region. Is my system is on connected to network? I think no need. I created a collection and synced with postman. But I don't know why this error came.

**To Reproduce**
1. Created collection with some REST API's
2. Created monitor for that to run every day at 5AM on Asia Pacific Region 
3. At 5 AM I got an error saying unable to load collection.
4. See attached screen shot for more info.

**Expected behavior**
Collection should be run automatically on scheduled time even when I was in sleep.

**Screenshots**
![screen shot 2018-06-01 at 10 31 50 am](https://user-images.githubusercontent.com/10599591/40821918-2df72aac-6587-11e8-914a-5f76c7d3dbed.png)

**App information (please complete the following information):**
 - App Type [e.g. Chrome App, Native App]
 - Postman Version [e.g. 6.0.1]
 - OS: [e.g. macOS High Sierra 10.13.2]

**Comment by thecuriousdev** [comment]
Hey @naveenkumarmallappa thanks for reporting this, are you still facing this issue? May be its because of the AWS outage that happened last week.

**Comment by naveenkumarmallappa** [comment]
Yes I am still facing issue. Collection is not running automatically. I changed time to 10 AM. but collection run successfully when my pc is on running otherwise it will get failed.
I attached screen shot for referral.

It showing error on JSON parser in test case. But it is all fine when I am executing collection runner manually.

![screen shot 2018-06-05 at 9 59 16 am](https://user-images.githubusercontent.com/10599591/40955414-842974e8-68a7-11e8-8697-a913c679b008.png)
